test/ios_test_runner_unit_test.sh

```diff
@@ -378,6 +378,73 @@ ios_unit_test(
 EOF
 }
 
+function create_ios_unit_tests_test_filter() {
+  if [[ ! -f ios/BUILD ]]; then
+    fail "create_sim_runners must be called first."
+  fi
+
+  cat > ios/test_filter_unit_test.m <<EOF
+#import <XCTest/XCTest.h>
+#import <XCTest/XCUIApplication.h>
+
+@interface TestFilterUnitTest : XCTestCase
+
+@end
+
+@implementation TestFilterUnitTest {
+  XCUIApplication *_app;
+}
+
+- (void)testPass {
+  XCTAssertEqual(1, 1, @"should pass");
+}
+
+- (void)testPass2 {
+  XCTAssertEqual(1, 1, @"should pass");
+}
+
+- (void)testPass3 {
+  XCTAssertEqual(1, 1, @"should pass");
+}
+
+@end
+EOF
+
+  cat > ios/TestFilterUnitTest-Info.plist <<EOF
+<plist version="1.0">
+<dict>
+        <key>CFBundleExecutable</key>
+        <string>TestFilterUnitTest</string>
+</dict>
+</plist>
+EOF
+
+  cat >> ios/BUILD <<EOF
+test_env = {
```

Comment:
can we drop this test_env if it's not required for this test?

Reply:
Oops some copy pasta is showing